src/Core/Generator/SchemaGeneratorFactory.cs

```diff
@@ -122,15 +122,13 @@ public static async Task<string> Create(RuntimeConfig config, string mode, int?
                 List<JsonDocument> dataArray = await schemaGeneratorSampler.GetSampleAsync();
 
                 logger.LogInformation("{0} records collected as Sample", dataArray.Count);
-                if (dataArray.Count == 0)
```

Comment:
Do we no longer error out when no data is present?

Reply:
we will error in later stage, when processing has been done for all the containers. ref. https://github.com/Azure/data-api-builder/blob/582a1156dae066ae98ca71319c6915f307b9689a/src/Cli/Exporter.cs#L120